code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -29,49 +26,17 @@
 		cell.use(amt) //Usage table: 1/tick if off/lowest setting, 4 = 4/tick, 6 = 8/tick, 8 = 12/tick, 10 = 16/tick
 	else
 		uneq_all()
-		stat = UNCONSCIOUS
-		update_headlamp(1)
+		low_power_mode = 1
+		update_headlamp()

```

Comment:
Borg without power are no longer unconscious but are in low_power_mode. It changes nothing gameplay wise, but it removed this weird case of being unconscious but not blind which is inconsistent with the rest of the code.